QuestPatcher.Core/ApkSigner.cs

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
  * All-C# APK signing code was taken from emulamer's Apkifier library: https://github.com/emulamer/Apkifier/blob/master/Apkifier.cs
```

Comment:
No longer true?

Reply:
The v1 signature part was, that comment can be updated.